recipes/folly/all/patches/0017-std-hash.patch

```diff
@@ -0,0 +1,180 @@
+diff --git a/folly/Uri-inl.h b/folly/Uri-inl.h
```

Comment:
Does this prevent the compilation of your code?

Reply:
You mean provided specialization of `std::hash` for `std::pair`/`std::tuple`? Yes. Because we already have these specializations in our project and they are widely used.